src/components/SearchBox/Jquery.SearchBox.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
+// @TODO - we can add this once jquery is removed
```

Comment:
Can you file a bug with this comment (it can be generic -- enable 'use strict' and remove TODO comments)